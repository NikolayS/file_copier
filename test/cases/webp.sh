#!/bin/bash

# to run this scenario, COPIERSERVICE should be defined (`export COPIERSERVICE=https://your.copier.host.name`)

http GET "$COPIERSERVICE/?wh=1&md5=1&src=https://lh3.googleusercontent.com/F_rxpKJFEclw3oN6W-ZCnGsu2hb0Tm5cE2FfCKVcdjnyBEosK8b39lVcZX9RFjDivq_4zLCSqT5P=w1920-h1080-rw-no" -h
