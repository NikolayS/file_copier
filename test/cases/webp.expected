HTTP/1.1 200 OK
X-File-Copier-Md5: 31bc6fd6fa2c39fc8834d86942859654
X-File-Copier-Img-Width: 622
X-File-Copier-Img-Height: 620
X-File-Copier-Size: 202313
X-Location: /data/93/e1/c3/2f/93e1c32fbe5e999d3157545b13f83a31584ac7b826871d19586239286667cc31.jpg
Access-Control-Expose-Headers: X-File-Copier-Size, X-Location, X-File-Copier-Img-Height, X-File-Copier-Img-Width, X-File-Copier-Md5

